demo/blockio/run.sh

```diff
@@ -227,8 +233,8 @@ fi
 # Summarize results
 SUMMARY_FILE="$OUTPUT_DIR/summary.txt"
 echo -n "" > "$SUMMARY_FILE" || error "cannot write summary to \"$SUMMARY_FILE\""
-first_speed=$(grep "^md5sum read speed:" $COMMAND_OUTPUT_DIR/0* | head -n 1 | awk '{print $4}')
-last_speed=$(grep "^md5sum read speed:" $COMMAND_OUTPUT_DIR/0* | tail -n 1 | awk '{print $4}')
+first_speed=$(grep "^md5sum read speed:" "$COMMAND_OUTPUT_DIR"/0* | head -n 1 | awk '{print $4}')
+last_speed=$(grep "^md5sum read speed:" "$COMMAND_OUTPUT_DIR"/0* | tail -n 1 | awk '{print $4}')
```

Comment:
Doesn't shellcheck complain about the enclosing $() not being in quotes ?

Reply:
Good point. I don't know why it didn't complain about those. Makes sense to add those, anyway.

Reply:
Double quotes added.